code/LeftAndMain.php

```diff
@@ -1755,6 +1755,21 @@ public function CMSVersionNumber()
         return $lockModules[$moduleName];
     }
 
+    /**
+     * Returns the minor version e.g. 4.10
+     * Will handle 4.10.x-dev by removing .x-dev
+     *
+     * @return string
+     */
+    public function CMSMinorVersionNumber(): string
```

Comment:
My gut feeling would be to just update `CMSVersionNumber` to drop the patch release version instead of creating a new method ... it's clearly only meant to be rendered in the template, and we don't use it anywhere else.

We also don't make any formal promise about what we'll be returning there beyond saying its a string.